public/app/features/alerting/unified/components/notification-policies/NotificationPoliciesList.tsx

```diff
@@ -218,34 +220,31 @@ export const NotificationPoliciesList = () => {
       )}
       {hasPoliciesData && (
         <Stack direction="column" gap={1}>
-          {rootRoute && (
```

Comment:
I don't really know why we were checking for `rootRoute` and why it was done twice 😅

Reply:
💯 `hasPoliciesData` includes the condition of having `rootRoute`